include/OpenColorIO/OpenColorIO.h

```diff
@@ -852,13 +852,18 @@ class OCIOEXPORT Config
      * \ref Config::validate will throw if the config does not contain
      * the matching display color space.
      */
+
+    bool viewIsShared(const char * dispName, const char * viewName) const;
+
     /// Will throw if view or colorSpaceName are null or empty.
     void addSharedView(const char * view, const char * viewTransformName,
                        const char * colorSpaceName, const char * looks,
                        const char * ruleName, const char * description);
     /// Remove a shared view.  Will throw if the view does not exist.
     void removeSharedView(const char * view);
 
```

Comment:
/// Clear all shared views. This will throw if any displays are still using the shared views.

Reply:
Sorry, I'm just checking - is this meant for clearSharedViews instead of viewIsShared?

Reply:
Yes, sorry, I put the comment on the wrong line.